### ISSUE TITLE
Implement ANC Medical History page for WCARO

### ISSUE THREAD

**Issue by cafootitt** [OpenSRP/opensrp-client-chw-anc#50]
The ANC medical history page is similar to Child medical history page, in that the View medical history row in the profile shows up only after the first services are recorded for the woman in the home visit.

![58843354-1f241c80-8640-11e9-91ed-0fde38904961](https://user-images.githubusercontent.com/3488986/58862841-40295380-86ba-11e9-8086-973bde741a9a.png)

@msschroeder Pls create a separate implementation issue for BA for medical history page, and then link it to this issue.

**Comment by cafootitt** [comment]
QA 10 July:

![Screenshot_20190709-163200](https://user-images.githubusercontent.com/3488986/60959400-a0359a00-a2f7-11e9-8c2e-1e24fd7af76d.jpg)

**Comment by cafootitt** [comment]
QA 1 August build (Togo apk):

English version:
![Screenshot_20191002-133658](https://user-images.githubusercontent.com/3488986/62459482-24106400-b788-11e9-9c0d-e242a43c37f3.jpg)


French version:
![Screenshot_20191002-134454](https://user-images.githubusercontent.com/3488986/62459390-e3b0e600-b787-11e9-8e9b-02e42d96914d.jpg)

**Comment by cafootitt** [comment]
Closing this and creating a new issue with remaining bugs.
